_layouts/default.html

```diff
@@ -75,5 +78,18 @@ <h2 class="appPrice">
                 </div>
             </div>
         </div>
+        <script>
+            function showConfirmationDialog() {
+                var isConfirmed = confirm("Direct download excludes getting automatic updates. Use Google Play or F-Droid instead. Press 'OK' to download anyway.");
```

Comment:
Talked to Izzy today, the IzzyOnDroid website now has instructions on getting updates too. So I think we can change this:

```suggestion
                var isConfirmed = confirm("Direct download excludes getting automatic updates. We recommend using IzzyOnDroid, F-Droid or Google Play instead. Press 'OK' to download anyway.");
```